bin/Data/Scripts/Editor/EditorSpawn.as

```diff
@@ -273,6 +282,13 @@ void PlaceObject(Vector3 spawnPosition, Vector3 normal)
         RefreshPickedObjects();
         return;
     }
+
+    if (parentToSelection && selectedNodes.length > 0)
+    {
+        Node@ selectedNode = selectedNodes[0];
+        if (selectedNode !is null)
```

Comment:
How selectedNodes[0] can be null?

Reply:
It might not always be null, but there's a similar check at Line 448 and I thought I would mimic that just in case.

Reply:
At line 448 selectedNode is null when selectedNodes.length == 0

Reply:
Should I remove the check then?

Reply:
yep

Reply:
Would it be okay if I made it into one statement? `selectedNodes[0].AddChild(spawnedObject);`

Reply:
yes